src/main/js/wiki/containers/Page.tsx

```diff
@@ -177,19 +180,23 @@ const Page: FC<Props> = (props) => {
   }
 
   return (
-    <div>
-      {wikiHeader}
-      <PageViewer
-        page={pageQuery.data}
-        wiki={wiki}
-        onDelete={deletePage}
-        onHome={pushLandingPageState}
-        onMove={onMove}
-        pagesLink={pagesLink}
-        historyLink={historyLink}
-        onRestore={onRestore}
-      />
-    </div>
+    <>
+      <div>
+        {wikiHeader}
+        <PageViewer
+          page={pageQuery.data}
+          wiki={wiki}
+          onDelete={deletePage}
+          onHome={pushLandingPageState}
+          onMove={onMove}
+          pagesLink={pagesLink}
+          historyLink={historyLink}
+          onRestore={onRestore}
+          currentBranch={branch}
+          branches={branchesQuery.data._embedded.branches}
```

Comment:
Missing error handling for `branchesQuery`

Reply:
Added Error handling but where can I find the file where the i18nKey are stored?

Reply:
Added the error handling as for the other querys and added translations